src/Xendit.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * Xendit.php
+ * php version 7.2.0
+ *
+ * @category Class
+ * @package  Xendit
+ * @author   Ellen <ellen@xendit.co>
+ * @license  https://opensource.org/licenses/MIT MIT License
+ * @link     https://api.xendit.co
+ */
+
+namespace Xendit;
+
+/**
+ * Class Xendit
+ *
+ * @category Class
+ * @package  Xendit
+ * @author   Ellen <ellen@xendit.co>
+ * @license  https://opensource.org/licenses/MIT MIT License
+ * @link     https://api.xendit.co
+ */
+class Xendit
+{
+    public static $apiKey;
+
+    public static $apiBase = 'https://api.xendit.co';
+
+    public static $libVersion = '2.0.0';
```

Comment:
Is it possible to read this from some dependency meta file (something like `package.json` for npm) so that we can always keep it on par with the actual version being published? I'm anticipating that people will miss this number out when trying to update our client library

Reply:
Yes, I've just added the version on the `composer.json`. Thanks for pointing it out!
Btw I've done the pre-commit also, sorry it took so long

Reply:
Cool cool. @ellennugroho can you load $libVersion from composer.json instead of using hard-coded value?

Reply:
sure, I've committed the changes

Reply:
btw what version should I put on the package? the old PHP client is version 2

Reply:
3 is good